Formula/akamai.rb

```diff
@@ -0,0 +1,26 @@
+class Akamai < Formula
+  desc "CLI toolkit for working with Akamai's APIs"
+  homepage "https://github.com/akamai/cli"
+  url "https://github.com/akamai/cli/archive/0.1.0.tar.gz"
+  sha256 "68a49db1c397ff55185b06252d1f2590395c6193566cdb52b2c910dd7d6c3889"
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    srcpath = buildpath/"src/github.com/akamai/cli"
+    srcpath.install buildpath.children
+
+    cd srcpath do
+      system "glide", "install"
+      system "go", "build", "-o", bin/"akamai", "."
+    end
+  end
+
+  test do
+    system "#{bin}/akamai", "list"
```

Comment:
Please have this do something and assert something. See `jose` and `vim` for some examples.